drools-osgi/drools-karaf-features/src/main/filtered-resources/repository/features.xml

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<features name="camel-drools-example-${project.version}"
```

Comment:
This is something I noticed by accident the other day -- the name should be something like `drools-${project.version}` (or probably better to use `drools-${version.org.kie}`). We need to fix this on master as well.